quantum-adapters/kiwoom-adapter/src/kiwoom_api/functions/market_info.py

```
@@ -0,0 +1,247 @@
+        # 삼성전자 시세표성정보 조회
+        test_data = {"stk_cd": "005930"}
+        
+        result = await fn_ka10007(test_data)
+        
+        print("=== ka10007 시세표성정보요청 테스트 ===")
+        print(f"응답코드: {result.get('Code')}")
+        print(f"헤더: {result.get('Header')}")
+        
+        if result.get('Code') == 200:
+            body = result.get('Body', {})
+            print(f"응답 데이터:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)